rakelib/prod/backfill_user_account_for_form_email_matches_profile_logs.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+desc 'Backill user account id records for FormEmailMatchesProfileLog'
+task backfill_user_account_for_form_email_matches_profile_logs: :environment do
+  def null_user_account_id_count_message
+    '[BackfillUserAccountForFormEmailMatchesProfileLog] FormEmailMatchesProfileLog with user_account_id: nil, ' \
+      "count: #{user_account_nil.count}"
+  end
+
+  def user_account_nil
+    FormEmailMatchesProfileLog.where(user_account_id: nil)
+  end
```

Comment:
Would it make more sense to make this a variable so the sql query can be stored and used throughout this file? Because it looks like now it's being called twice. If it's not an expensive call, maybe it's fine!

Reply:
It was calling 3 times total - count for log before the job, the query for the job itself, and count for log after the job. I refactored it so that the pre-job count & job query are the same to bring it down to just the before & after query.